web-ui/src/api/api.js

```diff
@@ -10,14 +10,19 @@ export const getAvatarURL = (email) =>
   "/services/member-profiles/member-photos/" +
   encodeURIComponent(email);
 
-let myAxios = null;
+function fetchAbsolute(fetch) {
+  return baseUrl => (url, ...otherParams) => url.startsWith('/') ? fetch(baseUrl + url, { credentials: 'include', ...otherParams }) : fetch(url, { credentials: 'include', ...otherParams })
+}
 
-export const getMyAxios = async () => {
-  if (!myAxios) {
-    myAxios = axios.create({
-      baseURL: BASE_API_URL,
-      withCredentials: true,
-    });
+let myFetch = null;
+
+export const getMyFetch = async () => {
+  if (!myFetch) {
+    myFetch = fetchAbsolute(fetch)(BASE_API_URL);
```

Comment:
Is there a synonymous param to 'withCredentials' for fetch to indicate usage of cookies?

Reply:
Yes, `credentials: 'include'`, and it's just a bit above on line 14 and is built into the fetch wrapper returned by `fetchAbsolute`.